README.md

```diff
@@ -254,6 +255,74 @@ plaidClient.getAccounts(access_token, (err, res) => {
 });
 ```
 
+## Payment Initiation
+
+Create payment recipient using IBAN and address without BACS
+
+```javascript
+const name =  'John Doe'
+const iban = 'NL02ABNA0123456789'
+const address = {
+    street: ['street name 999'],
+    city: 'London',
+    postal_code: '99999',
+    country: 'GB',
+};
+
+// Passing bacs as null
+plaidClient.createPaymentRecipient(name, iban, address, null, (err, res) => {
+    console.log(res.recipient_id)
+});
+```
+
+Create payment recipient using BACS with no IBAN or address
+
+```javascript
+const name =  'John Doe'
+const bacs = {
+    account: '26207729',
+    sort_code: '560029',
+}
+
+// For UK recipients, only bacs is required. iban and address are null
+plaidClient.createPaymentRecipient(name, null, null, bacs, (err, res) => {
+    console.log(res.recipient_id)
+});
+```
+
+Create payment
+
+```javascript
+const reference = 'testPayment'
+const amount = {
+    currency: 'GBP',
+    value: 100.00,
+};
+
+plaidClient.createPayment(recipient_id, reference, amount, (err, res) => {
+    console.log(res.payment_id)
+});
+```
+
+Create Link Token
```

Comment:
Is this actually specific to payment initiation? If so, should this text be updated to reflect that?

Reply:
Although creating a Link Token is not specific to Payment Initiation, I want to show a concrete example for this use case.

What do you think about changing it to "Create Link Token for Payment Initiation"? I'm not sure if it adds much, given that is already under the Payment Initiation section.

Reply:
+1 Yea, I think that would make it more clear that this is specific to payment initiation

Reply:
I think it's a good idea to point out that this example is specific to payment initiation